pkg/asset/installconfig/openstack/realvalidvaluesfetcher.go

```diff
@@ -1,30 +1,31 @@
-package validation
+package openstack
 
 import (
-	"github.com/pkg/errors"
```

Comment:
would like to keep the imports organized as
```
<stdlib>

<external>

<installer repo>

```

Reply:
Fixed.